gui/resources.xml

```diff
@@ -210,15 +211,15 @@ Vocabulary
           <!-- Code in document.py takes care of ltr/rtl details -->
           <signal name="activate" handler="undo_cb"/>
         </object>
-        <accelerator key="z"/>
+        <accelerator key="z" modifiers="GDK_CONTROL_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="Redo">
           <property name="icon_name">mypaint-redo-ltr-symbolic</property>
           <!-- Code in document.py takes care of ltr/rtl details -->
           <signal name="activate" handler="redo_cb"/>
         </object>
-        <accelerator key="y"/>
+        <accelerator key="z" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
```

Comment:
I rather stick with what the HIG suggests and leave the others(Z,Y, Ctrl+Y) as fallbacks. Regardless where the defaults are, all three sets are going to work in MyPaint out of the box. This will at least give the user the option to reassign the Z, Y, and Ctrl+Y if they wish without messing with the initial sets. I can at least add another phrase to the tooltips that Z, Y, and Ctrl+Y work as well so the user is aware of it.